packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -129,6 +128,39 @@ class FocusAttachment {
   }
 }
 
+/// Describe when should happen after [FocusNode.unfocus] is called.
+///
+/// See also:
+///
+///  * [FocusNode.unfocus], which takes this as its `disposition` parameter.
+enum UnfocusDisposition {
+  /// Focus the [FocusManager.rootScope]. This is often the least useful thing
+  /// to do with focus, since the root scope is effectively above any node that
+  /// appears in the widget tree, there is no context for it, and thus no
+  /// FocusTraversalPolicy that can be used to determine a next focus. This
+  /// means that if you hit TAB, you go nowhere.
```

Comment:
This seems really odd. If you hit TAB and there are notes that can receive focus, shouldn't the focus always go _somewhere_? Why's the root node special like this and outside of all traversal policies?